src/caretogether-pwa/src/Families/FamilyScreen.tsx

```diff
@@ -106,6 +108,34 @@ export function FamilyScreen() {
   useScreenTitle(family
     ? `${family.family?.adults!.filter(adult => adult.item1!.id === family.family!.primaryFamilyContactPersonId)[0]?.item1?.lastName} Family`
     : "...");
+  
+  enum ArrangementFilterOptionLabel {
+    Active = "Active",
+    Cancelled = "Cancelled"
+  }
+  const { arrangementFilterOptions, handleFilterArrangements } =
+    useFilterMenu(Object.keys(ArrangementFilterOptionLabel), [ArrangementFilterOptionLabel.Active]);
+  const meetsArrangementFilterCriteria = (arrangement: Arrangement): boolean => {
+    let result = false;
+    const selectedOptions = arrangementFilterOptions.filter(o => o.selected).map(o => o.text);
+    selectedOptions.forEach((option) => {
```

Comment:
This code could be simplified to just `const selectedOptions = ....filter(...).map(...).any(o => ...)` (so no `forEach` is needed).

Reply:
The reason to do this is that it would also get rid of the `let result = false` (avoiding mutable state) and the need for things like `break` statements (which can be hard to follow).

Reply:
Great suggestion - implemented! Also tweaked the UI to collapse when there were no matching results since I couldn't think of a valid use case for persisting empty placeholder space. 

To be clear, it already starts out collapsed if there aren't any results, but... prior to my latest update, if you add arrangements to the mix & then adjust the filters to filter them back out, Masonry was retaining whatever the last height was before the results went from a length of > 0 to a length of 0. This update makes it so that when there are 0 results, it ALWAYS collapses.

That said, I'm not _thrilled_ with the inline style tag I used to accomplish that UX improvement, so... if you have suggestions on a better approach there, I'm happy to take a pointer.